assembl/static2/js/app/components/common/glyphicon.jsx

```diff
@@ -1,22 +1,22 @@
+// @flow
 import React from 'react';
 import { I18n } from 'react-redux-i18n';
 
-class Glyphicon extends React.Component {
-  render() {
-    const glyph = this.props.glyph;
-    const color = this.props.color;
-    const size = this.props.size;
-    const desc = this.props.desc;
-    return (
-      <img
-        width={`${size}px`}
-        height={`${size}px`}
-        src={`/static2/img/icons/${color}/${glyph}.svg`}
-        alt={I18n.t(desc)}
-        title={I18n.t(desc)}
-      />
-    );
-  }
-}
+type Props = {
+  glyph: string,
+  color: string,
+  size: number,
+  desc: string
+};
+
+const Glyphicon = ({ glyph, color, size, desc }: Props) => (
+  <img
+    width={`${size}px`}
+    height={`${size}px`}
+    src={`/static2/img/icons/${color}/${glyph}.svg`}
```

Comment:
eventually change `/static2/img/icons/` to a const like `ICON_PATH` for instance ?

Reply:
yeahhhh old code... j'ai fait un gros refactoring pour supprimer la ref à static2. Il faut éviter de mettre des path en dur dans le code!!